### DISCUSSION TITLE
Create sensor that trigger dbt job B when dbt job A finished

### DISCUSSION THREAD

**Discussion by podviaznikov** [dagster-io/dagster#18606] [Q&A] [answered]
There is sensor API

```
@run_status_sensor(
    run_status=DagsterRunStatus.SUCCESS,
     monitored_jobs=[
         JobSelector(
             location_name="data_stack",
             job_name="job_a",
        )
     ],
    request_jobs=[JobSelector(
        location_name="data_stack",
        job_name="job_b",
    )]
   
)
```

But it's not clear how to specify `request_jobs` when the jobs were created using `build_schedule_from_dbt_selection`.

`request_jobs` needs `JobDefinition` but it's unclear how to do that from schedules `build_schedule_from_dbt_selection`.

**Comment by rexledesma** [comment, marked as answer]
`build_schedule_from_dbt_selection` returns a `ScheduleDefinition`. `ScheduleDefinition` has a `.job` property. You can feed that into `request_jobs`.

References:
- [`build_schedule_from_dbt_selection`](https://docs.dagster.io/_apidocs/libraries/dagster-dbt#dagster_dbt.build_schedule_from_dbt_selection)
- [`ScheduleDefinition`](https://docs.dagster.io/_apidocs/schedules-sensors#dagster.ScheduleDefinition)

**Reply by podviaznikov** [reply]
thank you! works!
